airgun/views/host_new.py

```diff
@@ -30,6 +31,29 @@ class DropdownWithDescripton(Dropdown):
     ITEM_LOCATOR = ".//*[contains(@class, 'pf-c-dropdown__menu-item') and contains(text(), {})]"
 
 
+class ItemList(Widget):
+    """Item list view, similar to table view, but using the ul/li structure"""
+
+    ROW = '//li/div[@class="pf-c-data-list__item-row"]/div'
+    COLUMN = './div'
+
+    def read(self):
+        """Return content of table"""
+        items = []
+        # make sure that parent locator works for str and also Locator object
+        parent_locator = (
+            self.parent.ROOT if type(self.parent.ROOT) == str else f'({self.parent.ROOT.locator})'
+        )
+        rows = self.browser.elements(f'{parent_locator}{self.ROW}')
+        for row in rows:
+            columns = []
+            elements = row.find_elements_by_xpath(self.COLUMN)
```

Comment:
Your Robottelo-Runner job on https://github.com/SatelliteQE/robottelo/pull/9996 is failing with `'WebElement' object has no attribute 'find_elements_by_xpath'`. Selenium recently removed the `find_elements_by_*` methods. [1] I checked out your branches locally and did not see the `find_elements_by_xpath` attribute on the `row` object here: 
```
(Pdb) dir(row)
['__abstractmethods__', '__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', '_abc_impl', '_execute', '_id', '_parent', '_upload', 'accessible_name', 'aria_role', 'clear', 'click', 'find_element', 'find_elements', 'get_attribute', 'get_dom_attribute', 'get_property', 'id', 'is_displayed', 'is_enabled', 'is_selected', 'location', 'location_once_scrolled_into_view', 'parent', 'rect', 'screenshot', 'screenshot_as_base64', 'screenshot_as_png', 'send_keys', 'shadow_root', 'size', 'submit', 'tag_name', 'text', 'value_of_css_property']
```
Could this possibly be the source of the error?

[1] https://stackoverflow.com/questions/72754651/attributeerror-webdriver-object-has-no-attribute-find-element-by-xpath

Reply:
I was wondering how it could be working on my local setup :sweat_smile: I have some ancient version of selenium
thanks, that helps a lot!